lib/readfile.rb

```diff
@@ -0,0 +1,31 @@
+
+def readFile(file)
+  return File.read(file)
+end
+
+
+def getWords(text)
+  text.gsub!(/[^A-Za-z ]/,'')
+  data = Hash.new
```

Comment:
Anytime you initialize a new object and then build it up is a good time to ask could I use `.reduce` or `.map`.

In this case, there's an even better option `.each_with_object`. Try refactoring and using it instead.
https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-each_with_object